test/tests/interfaces/pbs_sched_interface_test.py

```diff
@@ -0,0 +1,143 @@
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software - under
+# a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.interfaces import *
+from ptl.lib.pbs_ifl_mock import MGR_CMD_CREATE
+
+
+@tags('multisched')
+class TestSchedulerInterface(TestInterfaces):
+    """
+    Test suite to test different scheduler interfaces
+    """
+
+    def setUp(self):
+        # delete all schedulers except default
+        self.server.manager(MGR_CMD_DELETE, SCHED, id="@default")
+        TestInterfaces.setUp(self)
+
+    def test_create_successful_scheduler_object(self):
+        """
+        Creates a scheduler object succcessfully .
+        """
+        a = {'sched_cycle_length': 12000}
+        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
+        self.assertTrue(rc == 0)
+
+    def test_list_schedulers(self):
+        """
+        List a scheduler object.
+        """
+        pass
+
+    def test_delete_schedulers(self):
+        """
+        Delete a scheduler object succcessfully .
+        """
+        a = {'sched_cycle_length': 12000}
+        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
+        rc = self.server.manager(MGR_CMD_DELETE, SCHED, a, id="test1")
+        self.assertTrue(rc == 0)
+
+    def test_duplicate_scheduler_name(self):
+        """
+        Check for the scheduler object name.
+        """
+        a = {'sched_cycle_length': 12000}
+        rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
+        try:
+            rc = self.server.manager(MGR_CMD_CREATE, SCHED, a, id="test1")
+        except PbsManagerError, e:
+            self.assertTrue(
+                'qmgr: Error (15211) returned from server' in e.msg[1])
+
+    def test_permission_issue_on_sched(self):
+        """
+        Check for the permission to create/delete/modify scheduler object.
```

Comment:
Your test case says create/delete/modify but it tests only "create" part of the functionality